dashboard/src/components/PageHeader/PageHeader.scss

```diff
@@ -34,7 +34,9 @@
         h1 {
           padding-top: 0.2rem;
           margin: 0;
+          margin-right: 0.6rem;
           margin-left: 0.6rem;
+          font-size: 1.5rem;
```

Comment:
Reducing the h1 size, too large after the clr-ui update:

![4](https://github.com/vmware-tanzu/kubeapps/assets/11535726/baff8b2a-7053-41a0-b767-cae66feb9185)